homeassistant/components/zha/helpers.py

```diff
@@ -1106,7 +1106,6 @@ def async_cluster_exists(hass: HomeAssistant, cluster_id, skip_coordinator=True)
     return False
 
 
-@callback
 async def async_add_entities(
```

Comment:
As above, although it will need callers to be updated.

Reply:
> As above, although it will need callers to be updated.

Adjusted.

I have checked the zha code, and all callers are going through `async_dispatcher_connect` so I believe the callers do not need amending.

It is imported as `zha_async_add_entities`, and then used as partial:
```python
    unsub = async_dispatcher_connect(
        hass,
        SIGNAL_ADD_ENTITIES,
        functools.partial(
            zha_async_add_entities, async_add_entities, BinarySensor, entities_to_create
        ),
    )
```